src/main/java/com/google/api/codegen/DiscoveryFragmentGeneratorApi.java

```diff
@@ -115,6 +126,28 @@ protected void process() throws Exception {
     }
   }
 
+  /*
+   * Parses strings of the format
+   * "go:https://www.hello.com@java:https://www.world.com" and returns the value
+   * of the key corresponding to the key id.
+   */
+  private String parseAuthInstructionsUrl(String authInstructionsUrl, String id) {
```

Comment:
It's the same amount of work... I don't understand the benefit? The map is populated in the same for loop, all this does is not bother constructing a map because only one language matters.

Either way, this is temporary for this branch, I can change it on the other.